lib/insights/docs/components/buttons-menu.component.md

```diff
@@ -1,20 +1,11 @@
----
-Title: Buttons Menu Component
-Added: v2.4.0
-Status: Active
-Last reviewed: 2018-06-08
----
-
-# [Buttons Menu Component](../../../lib/core/src/lib/buttons-menu/buttons-menu.component.ts "Defined in buttons-menu.component.ts")
+# Buttons Menu Component
```

Comment:
Is it intentional to remove the Title, Added, Status and Last Reviewed metadata ?

Reply:
fixed, thanks